src/stories/molecules/layout/card.knobs.stories.js

```diff
@@ -6,5 +6,6 @@ export default () => ({
     image: object('Card - Image', data.card.image),
     title: text('Card - Title', data.card.title),
     description: text('Card - Description', data.card.description),
-    contentLink: object('Card - Link', data.card.contentLink)
+    contentLink: object('Card - Link', data.card.contentLink),
+    tags: data.card.tags.map((tag, i) => text(`Card - tag ${i}`, tag))
```

Comment:
Why do you use "object" for the contentLink?
They have to manage separate the text and the url.
Can be like this: 

contentLink: {
        text: text('Card - link text', data.card.contentLink.text),
        href: text('Card - link href', data.card.contentLink.href)
    }